test/AccountTest.java

```diff
@@ -0,0 +1,37 @@
+import org.junit.Before;
+import org.junit.Test;
+
+import  static org.junit.Assert.*;
+/**
+ * Created by ken-linux2 on 13/07/16.
+ */
+public class AccountTest {
+
+    private Account classUnderTest;
+
+    @Before
+    public void setUp(){classUnderTest=new Account();}
+
+    @Test
+    public void shouldReturnTheAccountNameThatWasSet(){
+        classUnderTest.setName("myAccountName");
```

Comment:
Try to extract constant to the beginning of the class and give them names with capital letters. It will make it easier to change their values in the future and the name will identify those parameters as constants.